src/settings/App.tsx

```diff
@@ -316,14 +345,56 @@ export class App extends Component<{}, AppState> implements OC.Plugin<OC.Search.
 	}
 }
 
+interface GroupSelectProps {
+        allGroups: Group[];
+	delegatedAdminGroups: Group[],
+        onChange: (options: {value: string, label: string}[]) => void;
+}
+
+function GroupSelect({allGroups, delegatedAdminGroups, onChange}: GroupSelectProps) {
+        const options = allGroups.map(group => {
+                return {
+                        value: group.id,
+                        label: group.displayname
+                };
+        });
+	
+        return <Select
+		onChange={(options) => { onChange && onChange(options) }
+		}
+		isMulti
+		value={delegatedAdminGroups.map(group => {
+			return {
+				value: group.id,
+				label: group.displayname
+			};
+		})}
+		className="delegated-admins-select"
+                options={options}
```

Comment:
Something general with this PR it seems to use a wild mix of tabs and spaces, could you make sure to always use tabs? :)

Reply:
done